lapce-core/src/editor.rs

```diff
@@ -1353,13 +1353,18 @@ impl Editor {
                 cursor.mode = CursorMode::Insert(selection);
                 vec![(delta, inval_lines, edits)]
             }
-            DeleteWordAndInsert => {
+            DeleteWordAndInsert | DeleteWordEndAndInsert => {
```

Comment:
This is the nicest way I could find to avoid code duplication, hopefully this is alright